client/src/utilities/Question.js

```diff
@@ -65,7 +65,9 @@ export function extractBonusText(bonus) {
 }
 
 export function extractActualAnswer(answer) {
-  const regMatch = answer.match(/<b>.*<\/b>/) || answer.match(/<strong>.*<\/strong>/);
+  //naively match the beginning of a question up to (but not including) the first left square brace
+  //if more powerful solution is needed, use lookahead like (?:(?!\[).)* and replace \[ as needed
+  const regMatch = answer.match(/^[^\[]*/);
```

Comment:
This is the only part of this PR that feels like a functionality change vs. a bug fix -- any thoughts on trying to look for the entirety of the answer rather than the known answerline, if it exists?

Pros: Known answerline is theoretically better than extracting extraneous things like e.g. pronunciation guides
Cons: Not all answers have known answerlines

This could theoretically be improved with a more complex function, I suppose? Copy/paste was a pretty quick feature, not one I spec'd out much.